debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4

```diff
@@ -449,6 +449,8 @@ DUPLICATE:                           'DUPLICATE';
 DYNAMIC:                             'DYNAMIC';
 ENABLE:                              'ENABLE';
 ENCRYPTION:                          'ENCRYPTION';
+ENCRYPTED:                           'ENCRYPTED';
```

Comment:
Add a new keyword let's say `ENCRYPTED_STRING_LITERAL` that would contain the string literal term, and it should work.

```ENCRYPTED_STRING_LITERAL:                           '`ENCRYPTED`' ```

Reply:
created as following one, hope its not overkill? just didn't want to left any variation out.
```
ENCRYPTED_STRING_LITERAL:            '`ENCRYPTED`' | '`encrypted`' | 'ENCRYPTED' | 'encrypted';
```

Reply:
Why can't we treat this in the same way we do the `engineName` rule?  There we define non-backtick values as Lexer literal values and then we also add `STRING_LITERAL` as a viable alternative for the rule.  The `STRING_LITERAL` would then cover the backtick equivalents and that becomes a parser implementation detail rather than one performed during the lexical analysis.